Source/WebApp-Service-Provider-DotNet/.config/dotnet-tools.json

```diff
@@ -9,7 +9,7 @@
       ]
     },
     "dotnet-ef": {
-      "version": "5.0.7",
+      "version": "6.0.0-rc.1.21452.10",
```

Comment:
Update this version by running ` dotnet tool update dotnet-ef --local` in the project directory